user-guide/operators.md

```diff
@@ -11,7 +11,7 @@ Ambassador Edge Stack allows developers to manage individual service/API deploym
 In more detail, Ambassador Edge Stack supports operations in the following ways:
 
 * Is simple to [deploy and operate](../../concepts/architecture), relying entirely on Envoy and Kubernetes for routing and scaling
-* Has extensive support for [TLS termination](../../user-guide/tls-termination) and redirects
+* Has extensive support for [TLS termination](../../user-guide/tls-termination) and [automatic HTTPS](/reference/host-crd) and redirects
```

Comment:
Shouldn't this be `../../reference/host-crd`?

Reply:
not anymore!